_studio/mfx_lib/vpp/src/mfx_vpp_sw_core.cpp

```diff
@@ -1111,8 +1114,6 @@ mfxStatus VideoVPPBase::Query(VideoCORE * core, mfxVideoParam *in, mfxVideoParam
             }
         }
 
-        MFX_CHECK_STS(MFX_ERR_UNSUPPORTED);
```

Comment:
Part of code was missing here after long transitions. You should add this line here:
swQuerySts = VideoVPP_SW::Query(core, out);
Otherwise check for swQuerySts is meaningless.

Reply:
SW VPP isn't supported